source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,125 +1,113 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: Fall back to the static mirrors
+title: GOV.UK CDN static mirrors
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-We maintain a static copy of most of the site, which gets used by the content delivery
-network (CDN) whenever origin (the application server) times out or serves an error
-response.
+GOV.UK uses the [Fastly content delivery network (CDN)](/manual/cdn.html) to handle most requests to [GOV.UK Origin](/manual/architecture-shallow-dive.html#a-user-visits-the-gov-uk-homepage). However, user requests will not get to Fastly in the following scenarios:
 
-This process is handled by our CDN config and is entirely transparent to us and
-our users. It happens multiple times a day, for lots of different reasons.
+- first and most commonly, user requests to GOV.UK may time out or return an error response
+- second, if Fastly goes down, user requests to Fastly will fail with a 5xx error code (the problem was caused by the server)
 
-This is why we refer to switching off Nginx on the origin cache machines as
-"falling back to the mirrors".
+When user requests to GOV.UK time out or return an error response, these requests instead go to a static copy of GOV.UK known as the GOV.UK mirror.
 
-## Hosting
+If Fastly has gone down, we [manually fall back to AWS CloudFront](/manual/fall-back-to-aws-cloudfront.html), the backup CDN. AWS then sends user requests to the GOV.UK mirror.
```

Comment:
Ah, I was sort of expecting this section to be changed a bit in light of https://github.com/alphagov/govuk-developer-docs/pull/3399#discussion_r796362313. Apologies if that wasn't clear.

It took me a while to figure out the purpose of these introductory paragraphs. I think it could be made clearer by becoming its own section:

```suggestion
The GOV.UK mirror is a static copy of GOV.UK.

## When is the static mirror used?

Most requests to GOV.UK are handled by the [Fastly content delivery network (CDN)](/manual/cdn.html), which sits between the user and [GOV.UK Origin](/manual/architecture-shallow-dive.html#a-user-visits-the-gov-uk-homepage). When a user requests a GOV.UK page, Fastly retrieves it from its cache, or fetches it from GOV.UK Origin if it does not have it in its cache.

Sometimes, GOV.UK Origin may time out or return a 5xx error response. When that happens, Fastly automatically fetches the page from the GOV.UK mirror instead.

Finally, if Fastly goes down, we'd need to manually [switch to AWS CloudFront](https://docs.publishing.service.gov.uk/manual/fall-back-to-aws-cloudfront.html) instead of Fastly. Where Fastly makes requests to GOV.UK Origin, AWS CloudFront instead makes all its requests to the static mirror.
```

We refer to 'GOV.UK mirror' and 'static mirror' interchangeably throughout this document. It might be best to stick with one (I usually hear it referred to as the latter, but might be worth making a Slack poll to gather a consensus from the developers).